script.js

```diff
@@ -1,4 +1,218 @@
+// define options for game mode
+var gameStateArray = [
+  "RollDiceModePlayerOne",
+  "ChooseDiceOrderPlayerOne",
+  "RollDiceModePlayerTwo",
+  "ChooseDiceOrderPlayerTwo",
+];
+
+// set game mode to Player One's Dice Rolls
+var gameMode = gameStateArray[0];
+
+// define variables for each die roll and the players' final values after ordering
+// for player one
+var diceRollOnePlayerOne = "";
+var diceRollTwoPlayerOne = "";
+var playerOneNumber = "";
+// for player two
+var diceRollOnePlayerTwo = "";
+var diceRollTwoPlayerTwo = "";
+var playerTwoNumber = "";
+
+// leaderboard function
+var playerOneWins = 0;
+var playerTwoWins = 0;
+var leaderboardText = "";
+var leaderboard = function () {
+  if (playerOneWins > playerTwoWins) {
+    leaderboardText = `Player 1 is in the lead with a score of ${playerOneWins}. Player 2 has a score of ${playerTwoWins}.`;
+  } else if (playerOneWins < playerTwoWins) {
+    leaderboardText = `Player 2 is in the lead with a score of ${playerTwoWins}. Player 1 has a score of ${playerOneWins}.`;
+  } else {
+    leaderboardText = `Both players are currently tied with a score of ${playerTwoWins}. Let's keep playing!`;
+  }
+  return leaderboardText + "<br>" + "Hit Submit again to keep playing!";
+};
+
+// dice roll function
+var diceRoll = function () {
+  diceOutput = Math.floor(Math.random() * 6) + 1;
+  return diceOutput;
+};
+
+// player one dice roll function
+var playerOneDiceRoll = function () {
+  diceRollOnePlayerOne = diceRoll();
+  diceRollTwoPlayerOne = diceRoll();
+  if (diceRollOnePlayerOne === diceRollTwoPlayerOne) {
+    outputValue =
+      // if both dice value are the same (player skips the ordering)
+      `🎲 Welcome Player 1 🎲` +
+      "<br>" +
+      ` You rolled ${diceRollOnePlayerOne} on both dice! Your number is ${diceRollOnePlayerOne}${diceRollOnePlayerOne}. Time for Player 2 to roll!`;
```

Comment:
i think that you can include the `<br>` in the backticks too probably just have to escape it with `/<br>` iirc